src/Compilers/CSharp/Test/Emit/Emit/CompilationEmitTests.cs

```diff
@@ -397,6 +397,34 @@ public void EmitMetadataOnly_DisallowOutputtingNetModule()
         }
 
         [Fact]
+        public void EmitMetadata_DisallowEmbedding()
```

Comment:
@tmat I'm having second thoughts about this specific case, which emits both a regular assembly and a ref assembly. It seems that we should allow embedding PDB information into the regular assembly in this case. What do you think?